django_filters/fields.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, field, lookup_choices, *args, **kwargs):
         }
         widget = LookupTypeWidget(**defaults)
         kwargs['widget'] = widget
+        kwargs['help_text'] = field.help_text
```

Comment:
> With this in place, then help_text defined for a filter will be displayed in the auto-generated documentation when lookup_expr is also provided. 

Gotcha. My concern was that the help_text would be rendered twice, although it looks the help_text may not have been rendered previously. Thanks.